config/manager/operator-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       serviceAccountName: yaks-operator
       containers:
         - name: yaks-operator
-          image: docker.io/citrusframework/yaks:0.4.0-SNAPSHOT
```

Comment:
The `-SNAPSHOT` gets automatically replaced in the release scripts. After the release the version should be updated with the next snapshot version. Can we keep the `-SNAPSHOT` here then?

Reply:
Am afraid not. The `controller-gen` and `operator-sdk` binaries deem this to be an illegal version. It might be it can be retained as part of the tag though.